pyproject.toml

```diff
@@ -24,10 +24,13 @@ gradio = "^4.20.1"
 huggingface_hub = "^0.21.4"
 click = "^8.1.7"
 wandb = { version = "^0.16.4", optional = true }
+posthog = "^3.5.0"
+python-dotenv = "^1.0.1"
 
 [tool.poetry.dev-dependencies]
 pre-commit = "^3.7.0"
 pytest = "^8.0.2"
+ipdb = "^0.13.13"
```

Comment:
remove? where is this being used here?

Reply:
same as below. these deps are required for dev work.